### DISCUSSION TITLE
Welcome to logseq-plugin-todo Discussions!

### DISCUSSION THREAD

**Discussion by ahonn** [ahonn/logseq-plugin-todo#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by RainSharky** [comment]
What are the "sun" and the arrow symbols for?

**Comment by vvieczorny** [comment]
ADD OPTION TO QUICK CAPTURE TASKS TO OTHER PAGES THAN JOURNAL!!
